templates/web/base/report/new/form_user.html

```diff
@@ -1,3 +1,4 @@
+<div id="private_form">
```

Comment:
Wonder if better to put this wrap in the parent fill_in_details_form.html, less likely to be overridden?

Reply:
equally though, it's harder to override, which was sort of my reasoning.